diagnostic.md

```diff
@@ -21,6 +24,12 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+
+    {{#link-to 'boston'}} Boston {{/link-to}}
+
+    {{#link-to 'campus'}} Back {{/link-to}}
+
+    Provides a way to travel to boston from campus, and then back to campus from Boston
```

Comment:
With a nested route we will be using:
`{{#link-to 'campus.boston'}}Boston{{/link-to}}`